docs/admin/out-of-resource.md

```diff
@@ -223,3 +318,42 @@ candidate set of pods provided to the eviction strategy.
 In general, it is strongly recommended that `DaemonSet` not
 create `BestEffort` pods to avoid being identified as a candidate pod
 for eviction. Instead `DaemonSet` should ideally launch `Guaranteed` pods.
+
+## Deprecation of existing feature flags to reclaim disk
+
+`kubelet` has been freeing up disk space on demand to keep the node stable.
+
+As disk based eviction matures, the following `kubelet` flags will be marked for deprecation
+in favor of the simpler configuation supported around eviction.
+
```

Comment:
we should discuss this in a design issue, its tangental to the doc.  what's documented is the current plan of record.